examples/website/src/shared/molecules/commentTemplate.tsx

```diff
@@ -202,6 +202,7 @@ export interface LoginToComment {
 
 export function ComposeComment(props: ComposeCommentProps) {
   const css = useStyle()
+  const maxCommentLength = 1000
```

Comment:
I suggest this to be from an environment variable. what do you think? @HanaAyman